test/Sentry.Unity.Editor.iOS.Tests/Sentry.Unity.Editor.iOS.Tests.csproj

```
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutDir>$(PackageEditorTestsPath)</OutDir>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="TestFiles/*.txt">
```

what about we organize files by unity version as folder. Will be handy with the xcodeproj and other such files:

```suggestion
    <None Update="TestFiles/2021.1/*.txt">
```